src/main/scala/dxWDL/compiler/GenerateIRTask.scala

```diff
@@ -106,18 +107,51 @@ case class GenerateIRTask(
   def apply(task: CallableTaskDefinition, taskSourceCode: String): IR.Applet = {
     Utils.trace(verbose.on, s"Compiling task ${task.name}")
 
+    def unwrapParamMeta(
```

Comment:
I think this should be a private method rather than a nested function since it is not using any other local variable and apply is not returning it.